File: 2017-03-28/Fragments/app/src/main/java/br/ufpe/cin/if1001/fragments/MainActivity.java
package br.ufpe.cin.if1001.fragments;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button b1 = (Button) findViewById(R.id.btn_exemplo1);
        b1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(getApplicationContext(),FragmentActivity.class));
            }
        });

        Button b2 = (Button) findViewById(R.id.btn_exemplo2);
        b2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(getApplicationContext(),DynamicFragmentActivity.class));
            }
        });
    }
}

File: 2019-04-08/Room/app/src/main/java/br/ufpe/cin/android/room/MainActivity.kt
package br.ufpe.cin.android.room

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    var estados: Array<Estado>? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        //O que acontece ao clicar no botão Adicionar Estado
        btnAdicionaEstado.setOnClickListener {
            startActivity(Intent(getApplicationContext(), AdicionarEstadoActivity::class.java))
        }

    }

}
